src/pages/index.js

```diff
@@ -32,79 +39,167 @@ const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
 const cardSelector = "#card-template";
-
 const cardTemplate =
   document.querySelector("#card-template").content.firstElementChild;
 const cardListEl = document.querySelector(".cards__list");
 const addNewCardButton = document.querySelector(".profile__add-button");
 
-// Initialize components
-const userInfo = new UserInfo(".profile__title", ".profile__description");
-
-const modalPreviewImage = new PopupWithImage("#image-modal");
-modalPreviewImage.setEventListeners();
-
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: (item) => {
-      const cardElement = renderCard(item);
-      section.addItem(cardElement);
-    },
-  },
-  ".cards__list"
-);
-section.renderItems();
+/* -------------------------------------------------------------------------- */
+/*                               Form Validators                              */
+/* -------------------------------------------------------------------------- */
 
 const profileFormValidator = new FormValidator(settings, profileEditModal);
 profileFormValidator.enableValidation();
 
 const addFormValidator = new FormValidator(settings, addCardModal);
 addFormValidator.enableValidation();
 
-// Event listeners
-profileEditButton.addEventListener("click", () => {
-  const user = userInfo.getUserInfo();
-  profileTitleInput.value = user.name;
-  profileDescriptionInput.value = user.description;
-  modalEditForm.open();
-});
+const editProfilePicFormValidator = new FormValidator(
+  settings,
+  editProfileImageModal
+);
+editProfilePicFormValidator.enableValidation();
 
-addNewCardButton.addEventListener("click", () => {
-  addFormValidator._disableSubmitButton();
-  addCardImageModal.open();
+/* -------------------------------------------------------------------------- */
+/*                        Class Insantiation Constants                        */
+/* -------------------------------------------------------------------------- */
+
+const userInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "675ffc74-7010-43d4-a4e4-01622ad8dbf1",
+    "Content-Type": "application/json",
+  },
 });
 
-function renderCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleImageClick);
+/* -------------------------------------------------------------------------- */
+/*                            Profile Picture Edit                            */
+/* -------------------------------------------------------------------------- */
 
-  return card.getView();
+function loadInitialProfilePicture() {
```

Comment:
Please, delete `loadInitialProfilePicture` because `loadInitialProfileData` gets all 4 params from the server (not only for the avatar). Please, set the avatar together with the name and about

You don't need to make the server busy 2 times with the same data